File: src/main/java/com/architecture_map/belarus/dto/ConstructionDto.java
package com.architecture_map.belarus.dto;

import lombok.AccessLevel;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

import java.util.HashSet;
import java.util.Set;

@RequiredArgsConstructor
@Getter
@Setter(AccessLevel.PUBLIC)
public class ConstructionDto {

    private String name;
    private AddressDto address;
    private Integer architecturalStyleId;
    private String buildingDate;
    private String description;;
    private Set<Integer> architectsId = new HashSet<>();
}

File: src/main/kotlin/by/architecture_map/belarus/dto/ConstructionDTO.kt
package by.architecture_map.belarus.dto

data class ConstructionDTO(
    var architecturalStyleId: String? = null,
    var region: String? = null,
    var district: String? = null,
    var buildingCenturyFrom: String? = null,
    var buildingCenturyTo: String? = null
)
